src/chapters/jdbc-auth.xml

```diff
@@ -170,7 +170,7 @@ Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
                     script located within the <filename>upgrade/</filename> directory:</para>
                 <informalexample>
                     <screen><prompt>$</prompt> <userinput>ls schema/upgrade/</userinput>
-<computeroutput>upgrade-pre-0.8.2.sql upgrade-pre-0.9.7.sql upgrade-pre-0.9.9.sql
+<computeroutput>upgrade-pre-0.8.2.sql upgrade-pre-0.9.7.sql upgrade-pre-0.9.10-incubating.sql
```

Comment:
Also, I believe the upgrade script is `upgrade-pre-0.9.10.sql` (and probably should stay that way for the sake of automation).